weave/mappers_python_def.py

```diff
@@ -66,9 +68,10 @@ def apply(self, obj):
         constructor_sig = inspect.signature(instance_class)
         for k, serializer in self._property_serializers.items():
             if serializer.type != OpDefType() and k in constructor_sig.parameters:
-                # None haxxx
-                # TODO: remove
-                obj_val = obj.get(k)
+                get_k = k
```

Comment:
would be nice to comment on this special case

Reply:
This hacking name stuff has all been centralized into the types layer and commented.